reconcile/change_owners/decision.py

```diff
@@ -11,38 +14,50 @@
 from reconcile.change_owners.diff import Diff
 
 
-@dataclass
-class Decision:
-
-    approve: bool = False
-    hold: bool = False
-
-
 class DecisionCommand(Enum):
     APPROVED = "/lgtm"
     CANCEL_APPROVED = "/lgtm cancel"
     HOLD = "/hold"
     CANCEL_HOLD = "/hold cancel"
 
 
+@dataclass
+class Decision:
+
+    approver_name: str
+    command: DecisionCommand
+
+
 def get_approver_decisions_from_mr_comments(
-    comments: list[dict[str, Any]]
-) -> dict[str, Decision]:
-    decisions_by_users: dict[str, Decision] = defaultdict(Decision)
+    comments: Iterable[dict[str, Any]]
```

Comment:
nit: would `Mapping` also work here (`Iterable[Mapping[str, Any]]`)?

Reply:
yes for sure. let me fix that

Reply:
fixed